### PR TITLE
fixed normalization

### PR DESCRIPTION
**Pull request by samnolen**
This small first contribution addresses the following issue.  https://github.com/blei-lab/edward/issues/752

In the computation of the loss function and gradients in KLpq.py, tensorflow.mean has been replaced by tensorflow.sum.  This removes a normalization which was unnecessary because the weights have already been normalized.

### CODE REVIEW

File: docs/tex/tutorials/klpq.tex [hunk 1]
```diff
@@ -85,7 +85,7 @@ \subsubsection{Adaptive Importance sampling}
     \sum_{s=1}^{S}
     \frac{p(\mathbf{z}_s \mid \mathbf{x})}{q(\mathbf{z}_s\;;\;\lambda)}
   \end{align*}
-  \item compute the weighted mean.
+  \item compute the weighted sum.
```
Comment (dustinvtran) [inline docs/tex/tutorials/klpq.tex]: You should also edit the appropriate equation, e.g., L125.

**Comment by dustinvtran** [comment]
LGTM
